tap_mysql/sync_strategies/binlog.py

```diff
@@ -129,6 +136,8 @@ def row_to_singer_record(catalog_entry, version, db_column_map, row, time_extrac
             the_utc_date = common.to_utc_datetime_str(val)
             row_to_persist[column_name] = the_utc_date
 
+        elif db_column_type == FIELD_TYPE.JSON and isinstance(val, dict):
```

Comment:
I'm wondering if we should only check `FIELD_TYPE.JSON` here. What if the data is a JSON column with a JSON array?

Reply:
Fixed!